UnitTests/MappingResolution/SqlStatementResolverTest.cs

```diff
@@ -316,32 +316,37 @@ public void ResolveSqlStatement ()
       Assert.That (fakeSqlStatement, Is.Not.EqualTo (setOperationCombinedStatement.SqlStatement), "This is important for the test below.");
 
       _stageMock
-          .Expect (mock => mock.ResolveSelectExpression (Arg.Is(constantExpression), Arg<SqlStatementBuilder>.Is.Anything, Arg.Is(_mappingResolutionContext)))
-          .Return (fakeExpression);
+         .Setup (mock => mock.ResolveSelectExpression (It.Is<Expression> (param => param == constantExpression), It.IsAny<SqlStatementBuilder>(), It.Is<IMappingResolutionContext> (param => param == _mappingResolutionContext)))
+         .Returns (fakeExpression)
+         .Verifiable ();
       _stageMock
-          .Expect (mock => mock.ResolveWhereExpression(whereCondition, _mappingResolutionContext))
-          .Return (fakeWhereCondition);
+         .Setup (mock => mock.ResolveWhereExpression(whereCondition, _mappingResolutionContext))
+         .Returns (fakeWhereCondition)
+         .Verifiable ();
```

Comment:
There are too many formatting details to resolve manually.
I think it would be a better approach to automatically format this file with the correct style settings.